README.md

```diff
@@ -0,0 +1 @@
+03.11 Initialized project
```

Comment:
We can keep history of changes here but I would also expect to see some instructions how to work with the project, scripts description (and whatever additional info may be needed at the later stages)